src/main/java/ortus/boxlang/runtime/jdbc/ExecutedQuery.java

```diff
@@ -206,7 +229,7 @@ public int getRecordCount() {
 		 */
 		Struct result = new Struct();
 		result.put( "sql", this.pendingQuery.getOriginalSql() );
-		result.put( "cached", false );
+		result.put( "cached", this.getIsCached() );
```

Comment:
I thought you would add the cache information here too
- cacheProvider
- cacheKey
- cacheTimeout
- cacheLastAccesTimeout

Reply:
Thanks for the reminder, yes I will take care of it.